app/apps/camunda/services.py

```diff
@@ -0,0 +1,105 @@
+import json
+
+import requests
+from django.conf import settings
+
+
+class CamundaService:
+    def get_process_definitions():
+        processes = []
+        request_url = settings.CAMUNDA_REST_URL + "process-definition"
+        response = requests.get(
+            request_url, headers={"content-type": "application/json"}
+        )
+
+        content = response.json()
+
+        for process in content:
+            processes.append(process)
+
+        return processes
+
+    def start_instance(process=settings.CAMUNDA_PROCESS_VAKANTIE_VERHUUR):
+        """
+        TODO: Use bussiness key instead of process key
+        """
+        request_url = (
+            settings.CAMUNDA_REST_URL + f"process-definition/key/{process}/start"
+        )
+        response = requests.post(
+            request_url, headers={"content-type": "application/json"}
+        )
+
+        content = response.json()
+
+        print(content)
+
+        return content["id"]
+
```

Comment:
Hier ga je er vanuit dat de request succesvol is en dat er een response is. Zou je wat veiliger op kunnen vangen. (ik krijg hier nu een error omdat mijn request faalt en content geen id veld heeft)